Lickability/PinpointKit
# Swift 3.0 Beta 5 Migration

## What It Does

Opened workspace in Xcode 8 Beta 5 and accepted Xcode’s recommended changes (again, apparently):

<img width="585" alt="screen shot 2016-08-12 at 11 35 56 am" src="https://cloud.githubusercontent.com/assets/7883805/17628098/c836ac4e-6081-11e6-9286-9efa1598e75e.png">

## How to Test

1. Check out the project
1. Build and run
1. Smoke test features

## Notes

- No source changes necessary from the Swift 3.0 migrator.
- This is based on the branch from https://github.com/Lickability/PinpointKit/pull/157, so please review / merge that first. There’s only one added commit here (790688a).
- This is pointed at `swift-3.0`, not `develop`.
